greedy_algorithms/jumpgame.cpp

```diff
@@ -0,0 +1,68 @@
+/**
+ * @file
+ * @brief Implementation of an algorithm to solve the jumping game problem [Click here for the problem](https://leetcode.com/problems/jump-game/)
```

Comment:
```suggestion
 * @brief Implementation of an algorithm to solve the [jumping game]((https://leetcode.com/problems/jump-game/)) problem
```

Reply:
note that this is an incorrect suggestion. there are additional parentheses in the suggested change.